src/base/0.0.0/end-no-respawn-fields.py

```diff
@@ -40,10 +41,23 @@ def migrate(cr, version):
     """
     )
 
+    ignored_fields = set()
+
+    for e in os.environ.get("suppress_upgrade_warnings", "").split(","):
+        if e.startswith("field:"):
+            ignored_fields.add(e[6:])
+
     for model, field, transient, store in cr.fetchall():
         qualifier = "field" if store else "non-stored field"
         if transient:
             qualifier = "transient " + qualifier
         lvl = util.NEARLYWARN if transient or not store else logging.CRITICAL
 
+        if "{}.{}".format(model, field) in ignored_fields:
+            ignore_fields |= field
+            util._logger.log(
+                util.NEARLYWARN, "Field respawn %s.%s explicitly ignored, skipping", field.model, field.name
+            )
+            lvl = util.NEARLYWARN
+
         _logger.log(lvl, "%s %s/%s has respawn!", qualifier, model, field)
```

Comment:
Bad copy/paste.
```suggestion
        action = ""
        if "{}.{}".format(model, field) in ignored_fields:
            lvl = util.NEARLYWARN
            action = "; explicitly ignored"

        _logger.log(lvl, "%s %s/%s has respawn%s.", qualifier, model, field, action)
```

Reply:
Yes actually I fixed it a little further so that the respawn signal is not the same as a basic field. This will allow to be more explicit.

Reply:
That doesn't mean that the proposed changed can't be applied.